src/main/java/com/eppo/sdk/helpers/AppDetails.java

```diff
@@ -0,0 +1,47 @@
+package com.eppo.sdk.helpers;
+
+import java.io.*;
+import java.util.*;
+
+public class AppDetails {
+
+    static AppDetails instance;
+    private String version;
+    private String name;
+
+    public static AppDetails getInstance () {
```

Comment:
I don't see where this is used - are you going to use it [here](https://github.com/Eppo-exp/java-server-sdk/blob/main/src/main/java/com/eppo/sdk/EppoClient.java#L189)?

Reply:
I missed pushing some code. We are using it in `EppoClient` class to get name and version.
Pushed the whole code